hphp/runtime/vm/jit/code-cache.cpp

```diff
@@ -97,7 +98,14 @@ CodeCache::CodeCache()
     exit(1);
```

Comment:
@mxw Calls can be PC-relative with the BL instruction in which case the range is also +/-128MB, or through a Register using the BLR instructions. 

We also have a few branch instructions with smaller immediates:

- `CBZ/CBNZ <Xt>, <label>` Compare and branch if register Xt is zero/non-zero to label. They have a +/- 1MB range. 

- `TBZ/TBNZ <Xt>, #<imm>, <label>` Test bit numbered by imm in register Xt with and branch to label. They have a +/- 32KB range.